x-pack/plugins/apm/server/lib/alerts/register_transaction_duration_alert_type.ts

```diff
@@ -157,7 +157,7 @@ export function registerTransactionDurationAlertType({
 
       const { agg } = response.aggregations;
 
-      const value = 'values' in agg ? agg.values[0] : agg?.value;
+      const value = 'values' in agg ? Object.values(agg.values)[0] : agg?.value;
```

Comment:
is this an unrelated change?

Reply:
Actually, this is a bug fix now that I think of it. This code (written by me of course) assumes that the response of a percentiles aggregation is `number[]`, but it's actually `Record<string, number>` where the key is the percentile. I'll verify whether that is the case and file a separate bug for it.

Reply:
https://github.com/elastic/kibana/issues/70770